source/material_model/simple.cc

```diff
@@ -246,15 +246,15 @@ namespace aspect
                                    "\\begin{align}"
                                    "  \\tau(T) &= H\\left(e^{-\\beta (T-T_0)/T_0}\\right),"
                                    "  \\qquad\\qquad H(x) = \\begin{cases}"
-                                   "                            \\tau_{min} & \\text{if}\\; x<\\tau_{min}, \\\\"
+                                   "                            \\tau_\\text{min} & \\text{if}\\; x<\\tau_\\text{min}, \\\\"
                                    "                            x & \\text{if}\\; 10^{-2}\\le x \\le 10^2, \\\\"
-                                   "                            \\tau_{max} & \\text{if}\\; x>\\tau_{max}, \\\\"
+                                   "                            \\tau_\\text{max} & \\text{if}\\; x>\\tau_\\text{max}, \\\\"
                                    "                         \\end{cases}"
                                    "\\end{align} "
```

Comment:
I ran this through a latex equation editor and the formatting looks correct. Only one comment - would it be worth replacing `x` in ` H(x)` to another variable so that it is not confused with the composition prefactor `xi`?

Reply:
You mean in reference to #1917? Let's make this a follow-up to the current patch. I like an incremental approach where each patch fixes one issue.

Reply:
I had not looked at #1917 yet, but that was indeed what the issue I had in mind. This aside, looks good.

Reply:
Great.

You know that you're allowed to merge patches now, right? ;-)

Reply:
See #1923.